open_academy/views/open_academy_session_views.xml

```diff
@@ -4,22 +4,63 @@
     <record model="ir.actions.act_window" id="open_academy_session_action">
         <field name="name">open.academy.session.action</field>
         <field name="res_model">open.academy.session</field>
-        <field name="view_mode">tree,kanban,form</field>
+        <field name="view_mode">tree,kanban,form,graph,calendar</field>
     </record>
 
     <record model="ir.ui.view" id="open_academy_session_view_tree">
         <field name="name">open.academy.session.view.tree</field>
         <field name="model">open.academy.session</field>
         <field name="arch" type="xml">
-            <tree>
+            <tree decoration-info="duration&lt;5" decoration-danger="duration&gt;15">
                 <field name="name" />
                 <field name="duration" />
                 <field name="number_seats" />
+                <field name="available_seats_computed" widget="progressbar" />
                 <field name="instructor_id" />
             </tree>
         </field>
     </record>
 
+    <record model="ir.ui.view" id="open_academy_session_view_graph">
+        <field name="name">open.academy.session.view.graph</field>
+        <field name="model">open.academy.session</field>
+        <field name="arch" type="xml">
+            <graph string="Asistance graph">
+                <field name="course_id" />
+                <field name="number_seats_taken" type="measure" />
+            </graph>
```

Comment:
The exercise says: Add a Graph view in the Session object that displays, for each course, **the number of attendees** under the form of a bar chart.

You can do a computed field to record the number of attedees_ids, if you have doubts you can ask me.

Reply:
Missing this